src/SprykerEco/Yves/Computop/Mapper/Init/PostPlace/PaydirektMapper.php

```diff
@@ -24,7 +24,7 @@ protected function createTransferWithUnencryptedValues(TransferInterface $quoteT
         $computopPaymentTransfer = new ComputopPaydirektPaymentTransfer();
         
         $computopPaymentTransfer->setCapture(ComputopConfig::CAPTURE_MANUAL_TYPE);
-        $computopPaymentTransfer->setTransId($this->generateTransId($quoteTransfer));
+        $computopPaymentTransfer->setTransId(substr($this->generateTransId($quoteTransfer), 0, 20));
```

Comment:
Please Move this logic of 20 symbols to `generateTransId`  function.
If you think it's likely to change in future, create separate function, still calling it inside `generateTransId` .
Also, move this 20 to module constants (not config in `config_default`).